packages/logger/src/Logger.ts

```diff
@@ -18,6 +18,30 @@ import type {
   PowertoolLogData,
 } from './types';
 
+/**
+ *
+ * # Logger class
+ *
+ * ## Intro
+ *
+ * The Logger utility provides an opinionated logger with output structured as JSON.
+ *
+ * ## Key features
+ *
+ * - Capture key fields from Lambda context, cold start and structures logging output as JSON
+ * - Log Lambda context when instructed (disabled by default)
+ * - Log sampling prints all logs for a percentage of invocations (disabled by default)
+ * - Append additional keys to structured log at any point in time
+ *
+ * ## Basic usage
+ *
+ * import { Logger } from "@aws-lambda-powertools/logger";
+ * const logger = new Logger();
+ *
```

Comment:
I see that you've put the link to the main docs, which is great, but maybe here there's a chance to uniform this section to the ones in Metrics & Tracer and expand it with a couple of basic code examples? What do you think?